doc/REPLICATED.md

```diff
@@ -0,0 +1,76 @@
+# Installing Waffle Takeout
+
+Please submit a request on [https://takeout.waffle.io](https://takeout.waffle.io) for more information on pricing and to register for a 45 day trial.
+
+Getting started guides and other Waffle.io documentation can be found at [here](https://github.com/waffleio/waffle.io/wiki).
+
+## Prerequisites
+
+* One dedicated host with Ubuntu 14.04 or greater installed
+* A Waffle.io Takeout license file
+* A GitHub Enterprise or GitHub.com application
+
+## Host Machine
+
+The standard setup consists of one host, running the following services:
+
+* Migrations
+* Poxa
+* Rally
+* Hedwig
+* Hooks
+* API
+* App
+* Nginx
+
+If you are using EC2 we recommend the c3.2xlarge instance types.
+
+For other setups we recommend hosts with 16 gigs of RAM and 8 CPUs.
+
+## Register a GitHub OAuth app
+
+Waffle Takeout talks to GitHub Enterprise and GitHub.com via OAuth. You will need to create an OAuth app on your GitHub Enterprise installation or GitHub.com that Waffle Takeout can connect to.
+
+The OAuth app registered will use the domain name pointing to your Platform host for the Homepage URL (e.g. https://waffle-io.your-domain.com).
+
+## Installation
+
+### Creating a Security Group
+
+If you're setting up your AMI for the first time you will need to create a Security Group. From the EC2 management console, create an entry for each port in the table below:
```

Comment:
This is specific to AWS, can we remove that language and just say your host machine needs to have these ports open?